src/ripple/nodestore/impl/DatabaseShardImp.cpp

```diff
@@ -728,162 +760,215 @@ DatabaseShardImp::onChildrenStopped()
 void
 DatabaseShardImp::import(Database& source)
 {
+    assert(init_);
```

Comment:
Because this function is a specific single-use, the source param can be locally created from `app_.getNodeStore()`, no need to pass it in.

Reply:
I forgot it's an override. I am okay with it as is.